src/game/handlers/explore.js

```diff
@@ -150,7 +150,7 @@ function sendExplorationStart (dispatch, _, msg, gameMap, char) {
     to: msg.chat,
     text: _('You started exploring %s!', gameMap.name),
     options: [
-      [`/explore_${gameMap.id}`],
+      [`/explore_${gameMap.id}`, ':information_source: /char_info'],
       stances.map(stance => `${stance.emoji} /stance_${stance.id}`),
```

Comment:
these explore routes are big :(, but i still think its better for smaller screens to use it there